src/main/java/tc/oc/pgm/commands/ReportCommands.java

```diff
@@ -0,0 +1,51 @@
+package tc.oc.pgm.commands;
+
+import app.ashcon.intake.Command;
+import app.ashcon.intake.parametric.annotation.Text;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.entity.Player;
+import tc.oc.component.Component;
+import tc.oc.component.types.PersonalizedText;
+import tc.oc.named.NameStyle;
+import tc.oc.pgm.api.Permissions;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.events.PlayerReportEvent;
+import tc.oc.util.components.Components;
+
+public class ReportCommands {
+
+  @Command(
+      aliases = {"report"},
+      usage = "<player> <reason>",
+      desc = "Report a player who is breaking the rules")
+  public static void report(MatchPlayer sender, Match match, Player player, @Text String reason) {
+    MatchPlayer accused = match.getPlayer(player);
+    match.callEvent(new PlayerReportEvent(sender, accused, reason));
+
+    final Component component =
+        new PersonalizedText(
+            Components.format(
+                "["
+                    + ChatColor.GOLD
+                    + "R"
+                    + ChatColor.WHITE
+                    + "] "
+                    + ChatColor.DARK_AQUA
+                    + "{0}"
+                    + ChatColor.YELLOW
+                    + " \u2794 "
+                    + ChatColor.DARK_AQUA
+                    + "{1}"
+                    + ChatColor.YELLOW
+                    + ": "
+                    + "{2}",
```

Comment:
I'm pretty sure this would be shorter if you used the text component stuff instead of string concat:
```java
new PersonalizedComponent(
  new PersonalizedComponent("["),
  new PersonalizedComponent("A", ChatColor.GOLD),
  new PersonalizedComponent("] "),
  sender.getStyledName(NameStyle.FANCY),
  new PersonalizedComponent(" \u2794 ", ChatColor.YELLOW),
...
```

Reply:
Changed to components.